presto-hive/src/main/java/io/prestosql/plugin/hive/HivePageSourceProvider.java

```diff
@@ -362,7 +362,7 @@ public int getIndex()
                     }
 
                     checkArgument(
-                            projectionsForColumn.computeIfAbsent(column.getBaseHiveColumnIndex(), HashSet::new).add(column.getHiveColumnProjectionInfo()),
+                            projectionsForColumn.computeIfAbsent(column.getBaseHiveColumnIndex(), columnIndex -> new HashSet<>()).add(column.getHiveColumnProjectionInfo()),
```

Comment:
is it the only place where we use `HashSet::new` like that?

Reply:
> is it the only place where we use `HashSet::new` like that?

It's the only one I could find, @sopel39.  I also checked a couple of other cases, such as `ArrayList::new`, but didn't find any instances.